asapdiscovery-data/asapdiscovery/data/state_expanders/state_expander.py

```diff
@@ -0,0 +1,51 @@
+import abc
+from enum import Enum
+
+from asapdiscovery.data.schema_v2.ligand import Ligand
+from pydantic import BaseModel, Field
+
+
+class StateExpanderType(str, Enum):
```

Comment:
We don't need this to be an enum its just extra complexity as this is only used to discern the type of expansion.